pkg/asset/ignition/bootstrap/content/bootkube.go

```diff
@@ -77,6 +77,7 @@ then
 		--asset-output-dir=/assets/kube-apiserver-bootstrap \
 		--config-output-file=/assets/kube-apiserver-bootstrap/config \
 		--config-override-files=/assets/bootkube-config-overrides/kube-apiserver-config-overrides.yaml
+		--cluster-config-file=/assets/manifests/cluster-config.yaml
```

Comment:
is this the config map? Then we should better use `--cluster-config-configmap-file`. Wdyt?